check_process

```diff
@@ -12,14 +16,10 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
+		#upgrade=1	from_commit=CommitHash
 		backup_restore=1
-		multi_instance=0
+		multi_instance=1
```

Comment:
Sooo this is changed from false to true here, but the manifest.json still has `"multi_instance": false`, hence why the multi-instance install fails